packages/carbon-graphs/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Changed
   * Fixed linter issues.
+  * Y-axis range getting changed when same data passed during reflow. 
```

Comment:
nit: this should be a statement of the code change, not the issue. 
e.g: `fixed Y-axis reflow bug`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR7).